Docs/contributing.md

```diff
@@ -0,0 +1,57 @@
+# Contributing to Vapor
+
+👋 Welcome to the Vapor team! We're happy to have you. Check out our [Code of Conduct](https://github.com/vapor/vapor/blob/master/Docs/code_of_conduct.md) if you haven't already, and join us on [discord](https://vapor.team) if you run into trouble.
```

Comment:
The line character count should be kept at 70 or less so it is easier to read git diffs.